### DISCUSSION TITLE
edge_topology

### DISCUSSION THREAD

**Discussion by xarthurx** [libigl/libigl#2004] [Q&A] [answered]
I'm writing wrapper func for some func in the `libigl`.

For the`edge_topology` func: https://github.com/libigl/libigl/blob/main/include/igl/edge_topology.cpp
It seems `V` is only used once for the conditional case (which seems not necessary), and not elsewhere.

So my question is, for this function, is the variable `V` really needed?

**Comment by alecjacobson** [comment, marked as answer]
Correct, but please use edge_flaps instead. edge_topology has been marked
as a duplicate for removal for a while.
-Alec
